src/lib/services/contentful/fetch.ts

```diff
@@ -0,0 +1,31 @@
+import { VERCEL, CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+
+// Re-evaluate utils using env vars on each request to ensure tests can
+//   properly mock and overwrite them.
+const contentfulConnected = (): boolean =>
+  !!(VERCEL === "1" && CONTENTFUL_SPACE_ID && CONTENTFUL_DELIVERY_API_TOKEN);
```

Comment:
you can basically infer the meaning of `VERCEL`, but maybe it's worth documenting, probably in the `.env` file, the [documented definition](https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables#system-environment-variables) of it

Reply:
Yeah the idea of using `VERCEL` here was that a contributor wouldn't have the Contentful id and key without also being able to run `vercel env pull` (so even though the doc says that "the app is deployed and running on Vercel" it's also `1` for us locally), but it's rather redundant since I still want to ensure that the Contentful id and key are defined. Will probably adjust this to remove the usage of `VERCEL`.

Reply:
Updated to remove redundancy in f652535!